tests/unit/test_predictor.py

```diff
@@ -169,7 +170,7 @@ def ret_csv_sagemaker_session():
     ims.sagemaker_client.describe_endpoint_config = Mock(return_value=ENDPOINT_CONFIG_DESC)
 
     response_body = Mock("body")
-    response_body.read = Mock("read", return_value=CSV_RETURN_VALUE)
+    response_body.read = Mock("read", return_value=bytes(CSV_RETURN_VALUE, "utf-8"))
```

Comment:
Reading a `botocore.response.StreamingBody` object will always return a bytes object, not a string.